app/Http/Controllers/api/v1/RoomController.php

```diff
@@ -120,65 +120,59 @@ public function start(Room $room, StartJoinMeeting $request)
         $meeting = $room->runningMeeting();
```

Comment:
The username in line 118 is used as the id, but in the comment of the user management pull request the username is not required. So it could be that the username is null and then it would lead to an error.